src/components/ec-donut/ec-donut.vue

```diff
@@ -101,92 +116,92 @@ export default {
 };
 </script>
 
-<style lang="scss">
-@import '../../scss/settings/colors/index';
-@import '../../scss/tools/';
-
+<style>
 .ec-donut {
-  display: flex;
-  flex-wrap: wrap;
+  @apply tw-flex;
+  @apply tw-flex-wrap;
 
   &__container {
-    margin-right: 24px;
+    @apply tw-mr-24;
+
     height: 108px;
 
-    @include media__up-to-sm {
-      width: 100%;
-      margin-right: 0;
-      display: flex;
-      justify-content: center;
+    @media screen and (max-width: theme('screens.sm')) {
+      @apply tw-w-full;
+      @apply tw-mr-0;
+      @apply tw-flex;
+      @apply tw-justify-center;
     }
   }
 
   &__legend {
-    @include h4;
-
-    line-height: 24px;
-    flex-wrap: wrap;
-    padding: 16px 0;
-    color: $level-3-body-and-headings;
-
-    @include media__up-to-sm {
-      padding: 16px 0 0;
-      display: inline-flex;
-      flex-direction: column;
-      margin: 0 auto;
+    @apply tw-h4;
+    @apply tw-flex-wrap;
+    @apply tw-py-16;
+    @apply tw-text-gray-3;
+
+    @media screen and (max-width: theme('screens.sm')) {
+      @apply tw-pt-16;
```

Comment:
Paddings can be on the same line
Margins can be on the same line
Flex classes can be on the same line